templates/zerver/help/editing-past-messages.md

```diff
@@ -0,0 +1,33 @@
+### Editing past messages
+
+ * If you want to edit a past message of your own after it has been
+   posted, hover your mouse over the message. You should see a
+   little pencil ![pencil](/static/images/help/pencil.png) at the end
+   of the message and a “down-chevron”
+   ![down chevron](/static/images/help/down_chevron.png) near the
+   upper right corner of the message. Clicking either one of these
+   will give you editing options.
+   
+ * It is possible that your chat is configured to set a limit on the time you may
+   edit a message, e.g. 10 minutes. As soon as that limit has passed, the little pencil
+   ![pencil](/static/images/help/pencil.png)  changes to a file icon 
+   <img src="/static/images/help/file.png" width = "18">. Depending on the configuration, clicking on it might allow you
+   to change the topic, 
+   like shown in this message 
+   
+   <img src="/static/images/help/message_time_limit_passed.png" width="480">
+
+ * You cannot entirely delete a message. When you remove all the text of a message and save it,
+   Zulip will automatically insert a "(deleted)" into your message. 
+   The fact that there was once a message there will still be
+   evident, and the time stamp of the original posting date and time
+   will still be shown.
+
+ * After you have edited a message, `(EDITED)` appears over the
+   message and to the right of your name. It is visible to everyone.
+
+ * Editing topic titles is discussed in a separate answer,
+   **[Edit topic](index.md#editing-a-topic-title)**.
+   
+**[Go back to “Posting and replying”](index.md#posting-and-replying)** |
+  **[Go back to “Table of contents”](index.md#table-of-contents)**
```

Comment:
From https://github.com/zulip/zulip/blob/master/templates/zerver/help/mention-a-team-member.md it appears that these links can be dropped